packages/shared/src/components/post/PostNavigation.tsx

```diff
@@ -58,6 +64,9 @@ function PostNavigation({
             className="rotate-90 btn-secondary"
             icon={<ArrowIcon />}
             onClick={onNextPost}
+            disabled={[PostPosition.Last, PostPosition.Only].includes(
+              postPosition,
+            )}
```

Comment:
Making this arrow (pointing towards the right) disabled must also consider whether it is the last item and whether you can still fetch more pages as these are paginated values. I think the variable is named `canFetchMore`.

Reply:
Makes sense